cmd/termui/presenter/chainInfoGetters.go

```diff
@@ -179,3 +179,13 @@ func (psh *PresenterStatusHandler) GetEpochInfo() (uint64, uint64, int, string)
 
 	return currentRound, epochFinishRound, epochLoadPercent, remainingTime
 }
+
+// GetAccountsSyncNodesProcessed will return information the number of trie nodes processed during sync
```

Comment:
// GetTrieSyncNumProcessedNodes will return the number of processed nodes during trie sync